bugbot/rules/not_landed.py

```diff
@@ -354,6 +354,13 @@ def get_bugs(self, date="today", bug_ids=[]):
             if not assignee:
                 continue
 
+            stack_graph = data.get("stackGraph", {})
```

Comment:
What is `stackGraph`? Do we have it in `data`?

Reply:
I wrote a separate test script that had the field `stackGraph`. After some testing I found that the field isn't present in data, so I'll make the necessary changes in the code to ensure that the `stackGraph` field is present.

Reply:
Added in cb1c47f8b462f8b3d08c5e3d2963af881618a381!